src/Makefile

```diff
@@ -4,17 +4,20 @@ CFLAGS=
 FC=gfortran
 FFLAGS=
 
-SOURCE_FILES=$(wildcard *.org)
+ORG_SOURCE_FILES=qmckl_context.org
```

Comment:
Could we use `$(wildcard qmckl_*.org)`, or is there a reason to explicitly put the file names ?

Reply:
We could, sure. Feel free to change (or I can do it if you want).